api/src/main/java/com/github/retrooper/packetevents/wrapper/play/server/WrapperPlayServerBossBar.java

```diff
@@ -0,0 +1,216 @@
+/*
+ * This file is part of packetevents - https://github.com/retrooper/packetevents
+ * Copyright (C) 2024 retrooper and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.github.retrooper.packetevents.wrapper.play.server;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import com.github.retrooper.packetevents.event.PacketSendEvent;
+import com.github.retrooper.packetevents.protocol.color.DyeColor;
+import com.github.retrooper.packetevents.protocol.packettype.PacketType;
+import com.github.retrooper.packetevents.wrapper.PacketWrapper;
+import net.kyori.adventure.bossbar.BossBar;
+import net.kyori.adventure.text.Component;
+
+public class WrapperPlayServerBossBar extends PacketWrapper<WrapperPlayServerBossBar> {
+
+    private UUID uuid;
+    private Action action;
+    private Component title;
+    private float health;
+    private BossBar.Color color;
+    private BossBar.Overlay overlay;
+    private List<BossBar.Flag> flags;
```

Comment:
this should be a `Set` (or even an `EnumSet`), as flags are not present multiple times
additionally, using a `Set` would be faster here

Reply:
I just made changes :)